src/base/base.pri

```diff
@@ -107,10 +107,10 @@ SOURCES += \
     $$PWD/rss/private/rssparser.cpp \
     $$PWD/utils/fs.cpp \
     $$PWD/utils/gzip.cpp \
+    $$PWD/utils/net.cpp \
```

Comment:
I'm really sorry for being such a nasty reviewer, but you misplaced this by one line, so move it one line downward, please (it has to be after misc.cpp, just like you did it with the header file).